src/ImageSequencer.js

```diff
@@ -237,6 +237,32 @@ ImageSequencer = function ImageSequencer(options) {
     return `sequencer -i [PATH] -s ${cliStringSteps} -d '${JSON.stringify(cliOptions)}'`
   }
 
+  // Checks if input is a string of comma separated module names
+  function detectSyntax(str) {
```

Comment:
If it returns boolean, should we name this `detectStringSyntax`? Or should we return "json" and "string"?

Reply:
Also, does this detect JSON saved as a string, AND native JSON objects in JS? Maybe we should specify! Thanks!

Reply:
I had the same thoughts

_Sent from my OnePlus3 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

Reply:
@jywarren fir now, this and `parseSyntax()` detect and handle normal comma separated strings AND JSON saved as strings. I mentioned above a possible improvement of supporting JSON modules as well:

> A suggestions, we can also make the `addSteps` method compatible with, say, a JSON module. If a user creates a JSON file of the steps and configurations and requires it inside addSteps, we should be able to handle that.

Would be a good addition, perhaps a follow up to this one?

Reply:
Yep that'd be great!